src/data/csv/cards.csv

```diff
@@ -0,0 +1,6 @@
+id,first_name,last_name,email,country,city,credit_card
```

Comment:
It should not be a part of `src` folder I guess.

Reply:
I agree, will be updated in next task